src/fragmentarium/ui/FragmentList.tsx

```diff
@@ -41,7 +41,9 @@ function FragmentList({
               <td key={index}>
                 {_.isFunction(property)
                   ? property(fragment)
-                  : fragment[property]}
+                  : property
+                      .split('.')
+                      .reduce((object, index) => object[index], fragment)}
```

Comment:
This allows us to pass `'script.period.abbreviation'` as string which is translated into `fragment.script.period.abbreviation`